libavfilter/dnn_filter_common.h

```diff
@@ -30,7 +30,9 @@ typedef struct DnnContext {
     char *model_filename;
     DNNBackendType backend_type;
     char *model_inputname;
-    char *model_outputname;
+    char **model_outputnames;
+    char *outputnames_expr;
```

Comment:
model_outputnames_expr, and we just use this var as the filter option, for a unified code later.

model_outputnames is always derived from model_outputnames_expr